pom.xml

```diff
@@ -51,7 +51,6 @@
 
   <properties>
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-      <jenkins.version>1.625.3</jenkins.version>
```

Comment:
Maybe it makes sense to keep it in order to avoid unexpected issues when the default version in Parent POM gets updated